### PR TITLE
Dact 514

### PR DESCRIPTION
**Pull request by rwalrondch**
Change to fix the mapping for the filing post close transaction. It used to auto map from office filing to FilingData used to pass to chips filing consumer.
Officer filing has altered, so we needed a new mapping for filingData.

### CODE REVIEW

File: src/main/java/uk/gov/companieshouse/officerfiling/api/model/mapper/FilingAPIMapper.java
Comment (PhilipGlover) [inline src/main/java/uk/gov/companieshouse/officerfiling/api/model/mapper/FilingAPIMapper.java:1]: there's a few unused imports in this class - Instant, ZoneId..

**Review by PhilipGlover — APPROVED** [review]
- I don't know the exact change to the filing data, but this looks ok
- tests pass
- approving now, though see comment re some unused imports

File: src/main/java/uk/gov/companieshouse/officerfiling/api/model/mapper/FilingAPIMapper.java
Comment (rwalrondch) [reply src/main/java/uk/gov/companieshouse/officerfiling/api/model/mapper/FilingAPIMapper.java:1]: unused imports deleted.

**Review by PhilipGlover — APPROVED** [review]
re-approving following last commit

**Pull request merged**